spec/lib/sufia/messages_spec.rb

```diff
@@ -31,30 +31,34 @@ class TestClass
   end
 
   describe "#single_success" do
-    let(:expected) { '<span id="ss-1"><a href="/files/1">File 1</a> has been saved.</span>' }
     it "should render a success message for a single file" do
-      expect(message.single_success(batch_id, single)).to eq(expected)
+      node = Capybara::Node::Simple.new(message.single_success(batch_id, single))
+      expect(node).to have_selector("span[id=\"ss-1\"]", text:"File 1 has been saved.")
```

Comment:
You don't need to change this here, but for future reference you could use single quotes here to prevent having to escape the double quotes.